self-tests.yml

```diff
@@ -373,6 +353,26 @@ tasks:
   - <<: *run-smoke-test
     name: smoke-test-endpoints
     tags: ["smoke"]
+  - name: generate-lint
```

Comment:
feels like we should have yaml anchors for generated tasks, mostly to demonstrate best(ish) practice.

Reply:
Done.